src/developer/web-api/route.md

```diff
@@ -18,70 +18,105 @@ X-Forwarded-User: <username>
 
 For the examples here we will be using the the [Echo API](https://learning.postman.com/docs/developer/echo-api/) from Postman which just returns what you send to it (including body in the case of `POST`).
 
-### GET Route without authentication
+### Running a route
+
+The following is an example of a JSON request creating a route:
 
 ```json
 {
-  "name": "postman-get",
-  "code": "postman-get",
+  "name": "postman",
```

Comment:
(minor) in order to make it clear that the code (not the name) can be used to access the route, let's make this something different
```suggestion
  "name": "Postman Echo",
```